website/content/v1.10/talos-guides/configuration/insecure.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "The insecure flag"
+description: "Learn how to use the insecure flag."
+---
+
+The `--insecure` flag is a per-command argument that allows you to communicate with the Talos API when a node is in maintenance mode, that is, before a node has been configured with a machine configuration or TLS certificates.
```

Comment:
This gives the wrong impression about the security.

`--insecure` in this context means "don't verify the server's TLS certificate". Even in maintenance mode API runs over TLS, but the identity of both client and server are not verified.

Probably this section should start with a note that regular Talos API uses mutual TLS verification, while in maintenance mode client doesn't present any certificate, while Talos (server) uses self-signed TLS certificate, which `--insecure` mode skips a check for.